DuggaSys/resulted.js

```diff
@@ -1156,16 +1156,15 @@ function onToggleFilter(colId) {
 	}
 
 }
-
 function exportCell(format, cell, colname) {
 	str = "";
 	if (format === "csv") {
 		if (colname == "FnameLnameSSN") {
-			if (cell.ssn.length > 11) {
+			if (cell.ssn.length > 0) {
 				str = cell.ssn + ";";
-			} else {
-				str = "19" + cell.ssn + ";";
-			}
+      }else{
+        str = "XXXXXXXX-XXXX";
```

Comment:
The real crux is that the `else` does not output a semicolon:
```suggestion
        str = "XXXXXXXX-XXXX;";
```
If that else ever ran, the csv would not have a terminating ";" for that column, and there would be a column mismatch.
Since it would take a missing ssn for this to occur, "this should never happen", but better safe than sorry I say.